clients/admin-ui/.eslintrc.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
     // causes bug in re-exporting default exports, see
     // https://github.com/eslint/eslint/issues/15617
     'no-restricted-exports': [0],
+    'import/prefer-default-export': 'off',
```

Comment:
allows us to directly function method here- https://github.com/ethyca/fidesops/pull/622/files#diff-adfcb02b29653b385133da876800f29f50b8c5bc39875695ebdf2324c63525f7R6

We also have other instances throughout admin-ui where we don't use default exports, so I'm unsure why this wasn't caught before.